### DISCUSSION TITLE
HomeyScript Reference

### DISCUSSION THREAD

**Discussion by gsommer** [community/community#19832] [Other Feature Feedback, Questions, & Ideas]
Hi,

is it possible to read the HomeyScript reference somewhere? The link in the readme of the docs folder does not work.

I am interested in buying Homey Pro, but before I want to know what kind of control scripting would give me. 

Kind Regards,
Gerhard

**Comment by Zi-Tao** [comment]
@gsommer https://athombv.github.io/com.athom.homeyscript/index.html

**Comment by gsommer** [comment]
Many thanks!
